community/sample-apps/react-relay/public/fonts/IM_Fell_Great_Primer_SC/OFL.txt

```diff
@@ -0,0 +1,93 @@
+Copyright (c) 2010, Igino Marini (mail@iginomarini.com)
+
+This Font Software is licensed under the SIL Open Font License, Version 1.1.
+This license is copied below, and is also available with a FAQ at:
+http://scripts.sil.org/OFL
+

```

Comment:
why are these things getting added to the code?

Reply:
I ported this over from the yelp clone tutorial; this is just styling for the header (app name).

Reply:
Can remove if you like

Reply:
removed